lib/puppetserver/ca/action/prune.rb

```diff
@@ -50,15 +50,24 @@ def run(inputs)
           loader = X509Loader.new(puppet.settings[:cacert], puppet.settings[:cakey], puppet.settings[:cacrl])
 
           puppet_crl = loader.crls.select { |crl| crl.verify(loader.key) }
-          prune_CRLs(puppet_crl)
-          update_pruned_CRL(puppet_crl, loader.key)
-          FileSystem.write_file(puppet.settings[:cacrl], loader.crls, 0644)
+          update_flag, dedupe_counter = prune_CRLs(puppet_crl)
+
+          if update_flag
+            update_pruned_CRL(puppet_crl, loader.key)
+            FileSystem.write_file(puppet.settings[:cacrl], loader.crls, 0644)
+            @logger.inform("Removed #{dedupe_counter} duplicated certs from Puppet's CRL.")
+            @logger.inform("Finished pruning Puppet's CRL.")
```

Comment:
Probably don't need this second log message now?

Reply:
Yeah I'm fine with removing it